readme.md

```diff
@@ -31,32 +33,76 @@ Don't see it? Try reinstalling Package Control. Altenatively, add the repository
 
 If it doesn't work, please file an issue.
 
+If you want contribute or use the latest commits simply git clone the repository `https://github.com/mrjoelkemp/Dependents` to your Packages directory with a directory name of `Dependents`. From sublime choose `Preferences -> Browse Packages`  to easily determine the path to the Packages directory for your install.  Be sure to remove the package via package control first if you had already done so to avoid package control updating the package.
+
+### Configuring settings
+
+There are two ways in which to invoke the package settings.  
+
+One is globally via entires in a user settings file `Preferences -> Package Settings -> Dependents -> Settings-User`.  You can open the `Settings-Default` file and copy and paste the contents as a starter template.   Do not edit the default file as that will be overwritten with updates.
+
+Alternatively you can define settings on a per-project basis by creating
+a `.deprc` file in the root of the sublime project/folder.  The paths you will specify are then **relative** to the sublime project/folder root. This is the preferred way if you are working on more than one project using sublime (aren't we all!)
+
+* `root`: (required) the ultimate/root path at which to limit dependent searching.  When using `.deprc` would be `""` if same as root of sublime project/folder.
```

Comment:
confirmed.  root is optional and ./ allows  root of dependents and root
in sublime window to work properly.
So just need to document this properly and you can close that issue.

 {
    "sass_root": "./"
  }

On Wed, Mar 11, 2015 at 7:24 PM, Joel Kemp <notifications@github.com> wrote:

> In readme.md
> <https://github.com/mrjoelkemp/Dependents/pull/131#discussion_r26274125>:
>
> > @@ -31,32 +33,76 @@ Don't see it? Try reinstalling Package Control. Altenatively, add the repository
> >
> >  If it doesn't work, please file an issue.
> >
> > +If you want contribute or use the latest commits simply git clone the repository `https://github.com/mrjoelkemp/Dependents` <https://github.com/mrjoelkemp/Dependents> to your Packages directory with a directory name of `Dependents`. From sublime choose `Preferences -> Browse Packages`  to easily determine the path to the Packages directory for your install.  Be sure to remove the package via package control first if you had already done so to avoid package control updating the package.
> > +
> > +### Configuring settings
> > +
> > +There are two ways in which to invoke the package settings.
> > +
> > +One is globally via entires in a user settings file `Preferences -> Package Settings -> Dependents -> Settings-User`.  You can open the `Settings-Default` file and copy and paste the contents as a starter template.   Do not edit the default file as that will be overwritten with updates.
> > +
> > +Alternatively you can define settings on a per-project basis by creating
> > +a `.deprc` file in the root of the sublime project/folder.  The paths you will specify are then **relative** to the sublime project/folder root. This is the preferred way if you are working on more than one project using sublime (aren't we all!)
> > +
> > +* `root`: (required) the ultimate/root path at which to limit dependent searching.  When using `.deprc` would be `""` if same as root of sublime project/folder.
>
> I also believe the root is now optional as well. I made it that way for
> more sass-centric devs to use dependents without worrying about JS
> configuration.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrjoelkemp/Dependents/pull/131/files#r26274125>.
>



-- 
Cheers, Cuidate, TTFN <http://www.youtube.com/watch?v=5Gu50vq5ux4>
David